data/environment.yml

```diff
@@ -0,0 +1,8 @@
+name: primer-trimming
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - biopython=1.79
+  - rdptools=2.0.3
```

Comment:
I think this environment also needs `snakemake` (provided in the `bioconda` channel) for the `### Setting up and activating an environment (optional)` section to be usable on its own.

Reply:
Ooh, maybe not. I just tried installing `snakemake` into the `primer-trimming` conda environment to no success. It looks like snakemake does not support Python 3.10 at the moment.
```
$ conda install -c bioconda snakemake
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: | 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                    

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - snakemake -> python[version='3.4.*|3.5.*|3.6.*|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0']

Your python: python=3.10

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.
```

Reply:
Hmm, that's weird. I have a Python 3.10 environment with snakemake in it (and the strange thing is that snakemake comes from the bioconda channel, like in your example, while Python 3.10.4 comes from conda-forge). I've updated the environment to include snakemake (and I've also renamed it), would you mind giving the install another try?

Reply:
Thanks, @jvkersch! That worked well!